lib/temporal.rb

```diff
@@ -3,6 +3,25 @@
 
 require 'temporal/connection'
 require 'temporal/version'
+require 'socket'
 
 module Temporal
+  CLIENT_NAME = 'temporal-ruby'.freeze
+
+  # TODO: Implement me
+  def self.build_id
+    Temporal::VERSION
+  end
+
+  def self.client_name
```

Comment:
We probably don't need a method for it, since it's an internally used name and not something we want the user to override. So might just keep the constant. Similar to `client_version`, which is already available via `Temporal::VERSION`